package.json

```diff
@@ -6,10 +6,10 @@
 	"private": true,
 	"workspaces": [
 		"dotcom-rendering",
-		"common-rendering"
+		"common-rendering",
+		"apps-rendering"
```

Comment:
This is coupling the apps a little further. Not that I mind, just making sure we know that's what we're doing.

Reply:
they were already coupled in this way i think, in that DCR's installation process called AR's installation process. 

This change mainly lets yarn take care of all npm packages in all instances. 

neither DCR or AR publish anything, so nothing should be being symlinked between the apps, afaik?

Reply:
Blammo. Now that you put it like that, they're less coupled.